tests/helpers/start-app.js

```diff
@@ -28,7 +30,48 @@ export default function startApp(attrs) {
   let attributes = Ember.merge({}, config.APP);
   attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
 
+  require.unsee('dummy/services/redux');
```

Comment:
I think the reason this wasn't working originally was because the requires are not returned to default (i.e. cleaned up) properly after the test.

Could you do something like this instead?
https://github.com/brettburley/ember-redux/commit/1309ad6142362b41908e73b0c297d4190592ff64

I think that's a bit cleaner to keep this require swapping outside of the main application boot in tests, and keeps the swapping isolated to that particular test.